docs/reference/deploy/master-config-reference.rst

```diff
@@ -1187,6 +1189,8 @@ those partitions/queues.
    the HPC partition named ``defq_GPU`` with the ``gpu_type`` property set, and Slurm constraint
    associated with the feature ``XL675d`` used to identify the model type of the compute node.
 
+.. _master-config-additional-resource-managers:
```

Comment:
you may want to rebase this against https://github.com/determined-ai/determined/pull/9050, since I already merged the commit that _reverts_ the multiRM docs changes into main